src/redisenterprise/setup.py

```diff
@@ -10,7 +10,7 @@
 from setuptools import setup, find_packages
 
 # HISTORY.rst entry.
-VERSION = '0.1.1'
+VERSION = '0.1.0'
```

Comment:
Please upgrade the version here and update `History.rst` file for new version's release notes

Reply:
Updated both files